scenario/__init__.py

```diff
@@ -1,6 +1,60 @@
 #!/usr/bin/env python3
 # Copyright 2023 Canonical Ltd.
 # See LICENSE file for licensing details.
+
+"""Charm state-transition testing SDK for Operator Framework charms.
+
+Write unit tests that declaratively define the Juju state all at once, define
```

Comment:
So we promote scenario as another way to do unit tests, then? We should sync with Charmcraft to make sure their profiles create a directory structure for tests that is aligned.

Reply:
We're (imminently) going to be promoting Scenario as the *preferred* way to do unit tests. I also tried to have these work if you were reading them when you'd done `pip install ops-scenario` but also if you had done `pip install ops[testing]` and were getting all the classes from the `ops.testing` namespace.

I do have a [ticket](https://warthogs.atlassian.net/browse/CHARMTECH-231) for updating the charmcraft profiles. I need to get all the bits in places first, but it's definitely in my definition-of-done.